mlflow/utils/model_utils.py

```diff
@@ -230,27 +232,26 @@ def _validate_path_exists(path, name):
         )
 
 
-def _validate_and_copy_model_code_and_config_paths(code_path, config_path, path):
-    """Copies the model code from code_path to a directory.
+def _validate_and_copy_file_path(file_path: Optional[str], path: str, name: str):
```

Comment:
Can we improve the naming of these arguments? I don't understand the difference between `file_path` and `path`. Also, can we handle the case where `file_path` is `None` elsewhere? It's a bit confusing for this method to operate on `None` and act as a no-op